Wi-Fi exporter/index.html

```diff
@@ -1,137 +1,166 @@
+    .description {
+      max-width: 800px;
+      margin: auto;
+    }
 
-</div>
-  
-<audio src=not wifi.mp3>
-  <audio srv=success. mp3></audio>
+    table {
+      border-collapse: collapse;
+      width: 100%;
+    }
 
-<table></table>
-  
-<script type=module>
-import {qr scanner} from './qe scanner. js'
+    th, td {
+      border: 1px solid #ddd;
+      padding: 8px;
+    }
 
-  
-  const button click= promise . with resolvers()
-  start. add event listen click{
-button click. resolve()
-  }
-  
-  
+    tr:nth-child(even) {
+      background-color: color-mix(in srgb, currentColor 5%, transparent);
+    }
 
-  function handle result({data}) {
-//data is a string 
-    if !data starts with "WIFI:"{
-      not wifi. current time = 0
-      not wifi. play()
-      return
+    video {
+      max-width: 100%;
+      display: none;
+    }
+  </style>
+</head>
+<body>
+  <div class="description">
+    <h1>Wi-Fi Password Exporter</h1>
+    <p>
+      On Android, you can go to <strong>Settings > Network & Internet > Internet > Saved networks</strong> to see a list of all Wi-Fi networks you've connected to. This tool allows you to export these networks and their passwords.
+    </p>
+    <ol>
+      <li>Click "Start Scanning".</li>
+      <li>Go to your saved networks on your phone.</li>
+      <li>Tap a network and click the "Share" button.</li>
+      <li>Show the QR code to the webcam until you hear a ding.</li>
+      <li>Repeat with all the networks.</li>
+    </ol>
+    <p>
+      The networks are saved locally, so you won't lose them if you accidentally close the browser. Once you're done, click the "Download as CSV" button.
+    </p>
+    <button id="start-scanning">Start Scanning</button>
+    <button id="download-csv">Download as CSV</button>
+  </div>
+
+  <video id="qr-video"></video>
+  <audio id="not-wifi-sound" src="Not Wi-Fi.mp3"></audio>
+  <audio id="success-sound" src="Success.mp3"></audio>
+
+  <table>
+    <thead>
+      <tr></tr>
+    </thead>
+    <tbody></tbody>
+  </table>
+
+  <script type="module">
+    import QrScanner from './QR scanner.js';
+
+    const startScanningButton = document.getElementById('start-scanning');
+    const downloadCsvButton = document.getElementById('download-csv');
+    const video = document.getElementById('qr-video');
+    const notWifiSound = document.getElementById('not-wifi-sound');
+    const successSound = document.getElementById('success-sound');
+    const table = document.querySelector('table');
+    const tableHead = table.tHead.rows[0];
+    const tableBody = table.tBodies[0];
+
+    let results = [];
+    const keyOrder = new Set();
+
+    function handleResult({ data }) {
+      if (!data.startsWith('WIFI:')) {
+        notWifiSound.currentTime = 0;
+        notWifiSound.play();
+        return;
+      }
+      successSound.currentTime = 0;
+      successSound.play();
+
+      const wifiData = data.substring(5).slice(0, -2).split(';').reduce((acc, part) => {
```

Comment:
Why not use replace like in the original instead of substring? 

Reduce is clever but I'd prefer if you used the for loop like in the original. And name the variable result